### PR TITLE
Update shortcut information for calc sheet in help tab

### PR DESCRIPTION
**Pull request by Darshan-upadhyay1110**
Signed-off-by: Darshan-upadhyay1110 <darshan.upadhyay@collabora.com>
Change-Id: I7ccbc194fe67fb36f9d68827dfa7c5636c9bc7e8

* Resolves: # 
* Target version: master 

### Summary
Added information in help tab (keyboard shortcuts)
 - About moving back and forth in between calc sheets using keyboard shortcuts
 
 
image

### CODE REVIEW

File: browser/html/cool-help.html [hunk 1]
```diff
@@ -144,6 +144,8 @@ <h2>Navigating in Spreadsheets</h2>
...
             <tr> <td class="function">Moves the cursor to the top edge of the current data range. If the row above the cell that contains the cursor is empty, the cursor moves up to the next row that contains data.</td> <td class="shortcut"><kbd>Ctrl</kbd><span class="kbd--plus">+</span><kbd>Up Arrow</kbd></td> </tr>
             <tr> <td class="function">Moves the cursor to the bottom edge of the current data range. If the row below the cell that contains the cursor is empty, the cursor moves down to the next row that contains data.</td> <td class="shortcut"><kbd>Ctrl</kbd><span class="kbd--plus">+</span><kbd>Down Arrow</kbd></td> </tr>
+            <tr> <td class="function">Moves from current sheet to next sheet.</td> <td class="shortcut"><kbd>Ctrl</kbd><span class="kbd--plus">+</span><kbd>Alt</kbd><span class="kbd--plus">+</span><kbd>Page Up</kbd></td> </tr>
+            <tr> <td class="function">Moves from current sheet to previous sheet.</td> <td class="shortcut"><kbd>Ctrl</kbd><span class="kbd--plus">+</span><kbd>Alt</kbd><span class="kbd--plus">+</span><kbd>Page Down</kbd></td> </tr>
```
Comment (pedropintosilva) [inline browser/html/cool-help.html:148]: Maybe better to have shorter wording and avoid using the verb move (to avoid misunderstandings about mocing the order of the sheets):

Go to next sheet.
Go to previous sheet.

**Review by pedropintosilva — CHANGES_REQUESTED** [review]

File: browser/html/cool-help.html (hunk 1 above)
Comment (Darshan-upadhyay1110) [reply browser/html/cool-help.html]: image
